chpl/chpl-service/src/main/java/gov/healthit/chpl/api/domain/ApiKey.java

```diff
@@ -2,54 +2,50 @@
 
 import java.io.Serializable;
 import java.util.Date;
+import java.util.Objects;
 
-import gov.healthit.chpl.util.Util;
+import javax.xml.bind.annotation.XmlTransient;
```

Comment:
Do these API-Keys show up in XML anywhere? I wouldn't have guessed that they would, which makes me wonder why they need to be marked with XmlTransient

Reply:
No they don't. I just copied it from somewhere else. Removed.